custom-routes-vis/src/main/webapp/script.js

```diff
@@ -281,15 +290,44 @@ function createRouteFromCoordinates(routeCoordinates, routeNum, totalDuration,
  * @param {num} routeNum The index of the selected route in the routes array.
  * @param {num} totalDuration The duration of the route in seconds.
  * @param {num} totalDistance The distance of the route in meters.
+ * TODO(carmenbenitez): Add in the route token returned from the Routes Preferred API.
```

Comment:
Wait why is this a TODO for me? I am not handling route tokens?

Reply:
Sorry I'll take that lol